### DISCUSSION TITLE
Shorten the LCD ribbon cables

### DISCUSSION THREAD

**Discussion by kdawg8** [sanni/cartreader#580] [Need help]
Hi, I'm building a V5 cart reader and the ribbon cables that came with the LCD are too large to stuff between the PCBs.  In the V5 build pictures it looks like the ribbon cables have been shortened.  I attempted to remove the crimp connector and broke it into about 5 pieces, could anyone help me with directions to remove the crimp connector on the other cable without breaking it?  I was able to attach the cable to the broken connector, but I'm not sure how stable it is.  Thanks!

**Comment by sanni** [comment]
It is true that they are super brittle and break into pieces most of the time.
I have now added some spare connectors to the buy list: https://www.aliexpress.com/item/32852909764.html

You can also use hot glue to attach the top to the bottom and ignore the broken off sides.
